ui/raidboss/data/triggers/o8n.js

```diff
@@ -6,110 +6,401 @@
       regex: /14:2929:Graven Image starts using Gravitational Wave/,
-      alertText: 'Get Right/East =>',
-      tts: 'right',
+      regexDe: /14:2929:Heilige Statue starts using Gravitationswelle/,
+      regexFr: /14:2929:Statue Divine starts using Onde Gravitationnelle/,
+      alertText: {
+        en: 'Get Right/East =>',
+        fr: 'Aller à Droite/Est =>',
+        de: 'Nach Rechts/Westen =>',
+      },
+      tts: {
+        en: 'right',
+        fr: 'Projection depuis le côté droit',
+        de: 'rechts',
+      },
     },
     {
       id: 'O8N Intemperate Will',
       regex: /14:292A:Graven Image starts using Intemperate Will/,
-      alertText: '<= Get Left/West',
-      tts: 'left',
+      regexDe: /14:292A:Heilige Statue starts using Unmäßiger Wille/,
+      regexFr: /14:292A:Statue Divine starts using Volonté Intempérante/,
+      alertText: {
+        en: '<= Get Left/West',
+        fr: '<= Aller à Gauche/Ouest',
+        de: '<= Nach Links/Westen',
+      },
+      tts: {
+        en: 'left',
+        fr: 'Projection depuis le côté gauche',
+        de: 'links',
+      },
     },
     {
       id: 'O8N Ave Maria',
       regex: /14:292B:Graven Image starts using Ave Maria/,
-      alertText: 'Look At Statue',
-      tts: 'look towards',
+      regexDe: /14:292B:Heilige Statue starts using Ave Maria/,
+      regexFr: /14:292B:Statue Divine starts using Ave Maria/,
+      alertText: {
+        en: 'Look At Statue',
+        fr: 'Regarder la statue',
+        de: 'Statue anschauen',
+      },
+      tts: {
+        en: 'look towards',
+        fr: 'Regarder la statue',
+        de: 'anschauen',
+      },
     },
     {
       id: 'O8N Indolent Will',
       regex: /14:292C:Graven Image starts using Indolent Will/,
-      alertText: 'Look Away From Statue',
-      tts: 'look away',
+      regexDe: /14:292C:Heilige Statue starts using Träger Wille/,
+      regexFr: /14:292C:Statue Divine starts using Volonté Indolente/,
+      alertText: {
+        en: 'Look Away From Statue',
+        fr: 'Ne pas regarder la statue',
+        de: 'Von Statue wegschauen',
+      },
+      tts: {
+        en: 'look away',
+        fr: 'Ne pas regarder la statue',
+        de: 'weckschauen',
+      },
     },
     {
       id: 'O8N Aero Assault',
       regex: /14:2924:Kefka starts using Aero Assault/,
-      infoText: 'Knockback',
-      tts: 'knockback',
+      regexDe: /14:2924:Kefka starts using Wallendes Windga/,
+      regexFr: /14:2924:Kefka starts using Méga Vent Véhément/,
+      infoText: {
+        en: 'Knockback on Boss',
+        fr: 'Projection depuis le boss',
+        de: 'Rückstoß vom Boss',
+      },
+      tts: {
+        en: 'knockback on boss',
+        fr: 'Projection depuis le boss',
+        de: 'Rückstoß vom Boss',
+      },
     },
     {
       id: 'O8N Flagrant Fire Single',
       regex: /1B:........:(\y{Name}):....:....:0017:0000:0000:0000:/,
+      regexDe: /1B:........:(\y{Name}):....:....:0017:0000:0000:0000:/,
```

Comment:
Just FYI, no need for these if they're duplicates.  If you specify a language like 'de', then it will use regexDe if it's there, otherwise it will fall back to regex on its own.  Would you mind removing all the 1B head marker duplicate lines you added?

Reply:
Ah good to know. That was my only worry there, that if I don't specify the regexDe it won't display anything when it's set to de. But if it falls back then I'll cut that back out again :)